webapp/channels/NOTICE.txt

```diff
@@ -1420,76 +1446,6 @@ SOFTWARE.
 
 ---
 
-## emoji-datasource
```

Comment:
What tools are you using to extract this information because this library is still being used.

Reply:
I check the libraries listed in this file https://github.com/mattermost/mattermost/blob/release-9.0/webapp/channels/package.json. 

I'll add it back now.